source/chapters/microphones.rst

```diff
@@ -0,0 +1,315 @@
+.. include:: /shortcuts.rstext
+
+.. _microphones:
+
+Using Microphones
+*****************
+There are three ways to use microphones with Mixxx, each with their own pros
+and cons:
+  * :ref:`Software monitoring<microphones-software-monitoring>`: Cheapest and 
+    simplest to set up, but there is a disorienting delay, referred to as 
+    "latency", between sound going into the microphone and hearing it come out 
+    speakers or headphones.
+  * :ref:`Direct monitoring<microphones-direct-monitoring>`: Recommended for 
+    most users. This is the easiest and cheapest to set up with good sound 
+    quality and without the latency issue of software monitoring.
+  * :ref:`Hardware mixers<microphones-hardware-mixers>`: Most expensive option 
+    to set up with high sound quality. This does not have the latency issue of 
+    software monitoring.
+
+.. warning:: USB microphones are not recommended. These devices have their own 
```

Comment:
This warning is to prominent for me. I think using a USB mic is a valid use case for broadcasting setups. I think it is quite common to have USB headsets. 
There is not much a difference between plugin a mic into a soundcard or using an USB mic.
The only issue is that the soundcard might be poor.
It would be better to tell here how to configure an USB headsets.

Reply:
I think this warning is as prominent as it needs to be. For similar costs as USB microphones users can get a soundcard capable of direct monitoring and a regular microphone.

Reply:
That is not true. There are real cheap usb headsets available < 20 €
There are also user we which have already an USB headset.

Reply:
I am doubtful users who want to use a headset with Mixxx will bother to read the manual before buying hardware or care what it recommends. Those are not the users who this warning is for.Typical USB microphones cost $100-150. For a similar price a user could get a Behringer U-Phoria UMC22 and a decent microphone that would let them use direct monitoring.

Reply:
You are fully right, If a user wants to spend a reasonable amount of money, he should not go with a USB headset solution. But there is no big disadvantage, using an existing headset for broadcasting, if the own voice is passed only to the sidechain. 
So we just need a sentence that expresses this more differentiated.
For a life setup a cheep headset is almost useless.

Reply:
I will clarify this in the Software Monitoring section, but I don't think this warning box should have any more detail.